src/packages/core/src/external/everyviewAnalytics.tsx

```diff
@@ -19,6 +19,7 @@ export const EveryviewAnalytics = (props: IEveryviewAnalyticsProp): null => {
     }
 
     // NOTE(krishan711): figerprintjs references window immediately: https://github.com/fingerprintjs/fingerprintjs/issues/602
+    // eslint-disable-next-line
```

Comment:
you should disable the specific error only (you are currently disabling all errors for the next line